Proposal: Culture Context (#329)
getsentry/develop: src/docs/sdk/event-payloads/contexts.mdx

  }
}
```

> ## Current Culture Context

Reviewer: This "current culture" is a .NET thing because you can have a different culture set on a thread. So the device culture can be one, and the thread another. We could simplify here (and/or reword it) if that makes sense.

Reviewer 2: @bruno-garcia would be possible then that you are sending 2+ current_culture values? per thread for example?

Reviewer: .NET currently sends two (if they differ). `Culture` and `Culture Culture`. With this format as key :hides:

But if this lands in the spec we'd be adjusting accordingly

Reviewer 3: Contexts per thread is a concept we don't really support, but you can send several contexts with different keys.

> ## Current Culture Context

Reviewer 2: maybe it'd make sense to move some stuff out from the device context to this one then, like `timezone`

Reviewer 2: as timezone now belongs to culture as well, lets deprecate the one in the device context?

Author: See 
- https://github.com/getsentry/develop/pull/329#issuecomment-836457964
- https://github.com/getsentry/develop/pull/329#issuecomment-836485242

Reviewer 2: deprecating is more like, adding a note that this is also available under the new culture type, not really removing.

Current Culture Context describes certain properties of the culture in which 
the software is used.

> The `type` and default key is `"current culture"`.

Reviewer 2: yep, thats the standard already, lets stick to it

`calendar`

: _Optional_. For example `GregorianCalendar`. Free form string. 

`displayName`

: _Optional_. Human readable name of the current culture. For example `English (United States)`

`locale`

> : _Optional_. Two letter code of the current culture. For example `en-US`

Reviewer: In .NET it's:

> The name is a combination of an ISO 639 two-letter lowercase culture code associated with a language and an ISO 3166 two-letter uppercase subculture code associated with a country or region.
```suggestion

So maybe we change to:
: _Optional_. The name identifier, usually following the RFC 4646. For example `en-US` or `pt-BR`.
```

Reviewer 2: already exists under `device.language`, we'd have to move it

Reviewer 2: @ueman what's about deprecating `device.language` and moving to culture? I guess it'd make sense

Author: See 
- https://github.com/getsentry/develop/pull/329#issuecomment-836457964
- https://github.com/getsentry/develop/pull/329#issuecomment-836485242

> `24hourFormat`

Reviewer 2: wondering if in another language, that are different formats other than 12-24, guess not ^^

  }
}
```

## Culture Context

> Current Culture Context describes certain properties of the culture in which 

Author: ```suggestion
Culture Context describes certain properties of the culture in which 
```

the software is used.

> The `type` and default key is `"current_culture"`.

Reviewer 3: Please use `type: culture` here, since that is the true type of the context. You can still have a `current_culture` or other key for SDK-specific conventions.

`calendar`

: _Optional_. For example `GregorianCalendar`. Free form string. 

`display_name`

> : _Optional_. Human readable name of the culture. For example `English (United States)`

Author: `display_name` should probably be inferred from `local`. So if `local` is `en-US`, `display_language` should be `English (United States)`.

> : _Optional_. Human readable name of the current culture. For example `English (United States)`

Author: This could be inferred from the local identifier.

> : _Optional_. Human readable name of the current culture. For example `English (United States)`

Reviewer 2: ```suggestion
: _Optional_. Human readable name of the culture. For example `English (United States)`
```

> `locale`

Reviewer 2: aka device.language

: _Optional_. The name identifier, usually following the RFC 4646. For example `en-US` or `pt-BR`.

`is_24_hour_format`

: _Optional_. boolean, either `true` or `false`.

> `timezone`

Reviewer 2: aka device.timezone
